pkgs/development/libraries/gdal/default.nix

```diff
@@ -271,6 +267,9 @@ stdenv.mkDerivation (finalAttrs: {
     # flakey tests, to remove on next release
     "test_vsiaz_write_blockblob_chunk_size_1"
     "test_vsiaz_fake_write"
+    # failing for unknown reason
+    # https://github.com/OSGeo/gdal/pull/10806#issuecomment-2362054085
+    "test_ogr_gmlas_billion_laugh"
```

Comment:
This looks like a DoS attack mitigation - are there any GDAL/Xerces-C experts in the Nix community who might be able to help?

Reply:
Not sure.